sdk/textanalytics/ai-text-analytics/samples/javascript/beginAnalyzeBatchActions.js

```diff
@@ -37,42 +37,48 @@ async function main() {
   const resultPages = await poller.pollUntilDone();
 
   for await (const page of resultPages) {
-    const keyPhrasesResults = page.extractKeyPhrasesResults[0];
-    for (const doc of keyPhrasesResults) {
-      console.log(`- Document ${doc.id}`);
-      if (!doc.error) {
-        console.log("\tKey phrases:");
-        for (const phrase of doc.keyPhrases) {
-          console.log(`\t- ${phrase}`);
+    const keyPhrasesAction = page.extractKeyPhrasesResults[0];
```

Comment:
NIT: do u think it is worth calling out that  because there was only one action of this type you could do `[0]` instead of printing the content in a for loop?

Reply:
Ok I got rid of the loop.